client/app/src/accounts/account.service.js

```diff
@@ -971,6 +801,9 @@
 
       fetchAccountAndForget: fetchAccountAndForget,
 
+      // return a copy of the object, so the original can't be changed
+      defaultFees: JSON.parse(JSON.stringify(self.defaultFees)),
```

Comment:
Probably a better option is freezing the object (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze). It's something that we should start using on more places.